pychunkedgraph/backend/graphoperation.py

```diff
@@ -752,7 +752,7 @@ def __init__(
         multicut_as_split: bool,
     ) -> None:
         super().__init__(cg, user_id=user_id)
-        log_record = cg.read_log_row(superseded_operation_id)
+        log_record = cg.read_log_row(superseded_operation_id)[0] #TODO I don't know why this made it work...?
```

Comment:
The `read_log_row()` function return type was changed in 3d5bfaeadf2cf185887aacaed03a4c8c282a9d3f to additionally include a timestamp. This likely also broke other calls - at least the `RedoOperation.__init__`.

Reply:
The tests are failing for the same reason - I only added a mock for `read_log_row` back then, so now that one needs to be changed as well to include the time stamp.

Reply:
I just pushed a change adding the `[0]` de-timestamping to the other uses of `read_log_row()` in `graphoperation.py`. Should I change the tests too?

Reply:
That would be great, yes! Improvements and additions to the tests are also always welcome, but at least the few that we have should remain working.